middle_end/flambda2/from_lambda/closure_conversion.ml

```diff
@@ -3548,7 +3548,7 @@ module CIS = Code_id_or_symbol
 module GroupMap = Numbers.Int.Map
 module SCC = Strongly_connected_components.Make (Numbers.Int)
 
-let bind_code_and_sets_of_closures all_code sets_of_closures acc body =
+let bind_code_and_static_consts all_code sets_of_closures acc body =
```

Comment:
You could call the function `bind_symbols` and only take `acc` as parameter (the `all_code` and `sets_of_closures` field are retrieved from `acc` at the only call site). Alternatively, you could take the `declared_symbols` as parameter to match `all_code` and `sets_of_closures`.